app/models/ticket_stakeholder.rb

```diff
@@ -9,6 +9,14 @@ class TicketStakeholder < ApplicationRecord
   belongs_to :ticket
   belongs_to :stakeholder, polymorphic: true
 
+  scope :belongs_to_user, lambda { |user_id|
+    where(stakeholder_id: user_id)
```

Comment:
This should require an additional `where(stakeholder_type: 'User')` similar to the `belongs_to_groups` scope below, doesn't it?

Reply:
Oh I had it initially TBH, I don't know how it disappeared 😅
Added it now.